compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingTypes.h

```diff
@@ -83,6 +83,15 @@ struct MatmulNarrowDim {
   bool isN() const { return dim == Dim::N; }
 };
 
+struct MatmulDims {
+  int64_t m;
+  int64_t n;
+  int64_t k;
+};
```

Comment:
Do we just use `linalg::ContractionDimensions`?

Reply:
I am not sure what you mean? To use `linalg::ContractionDimensions` instead with a single element in every member variable?

Reply:
> To use linalg::ContractionDimensions instead with a single element in every member variable?

Yes, but let's clarify what is encoded in `iteration_sizes` first.